sfepy/discrete/fem/fields_base.py

```diff
@@ -236,10 +242,6 @@ def __init__(self, name, dtype, shape, region, approx_order=1):
         Assumes one cell type for the whole region!
         """
         shape = parse_shape(shape, region.domain.shape.dim)
-        if not self._check_region(region):
-            raise ValueError('unsuitable region for field %s! (%s)' %
-                             (name, region.name))
-
         Struct.__init__(self, name=name, dtype=dtype, shape=shape,
```

Comment:
Does it mean all regions are now usable? Or just cell and facet ones?

Reply:
Yes, just cell and facet regions are usable. Fixed in `._setup_geometry()`.